openstackgit-update.yml

```diff
@@ -33,22 +33,13 @@
         path: "{{ working_dir }}/openstackgit"
         state: directory
 
```

Comment:
The git clone of openstack-ansible is being done in the jenkins job itself. See https://github.com/rcbops/jenkins-rpc/pull/727